exercise_10/index.js

```diff
@@ -0,0 +1,11 @@
+import transformCollectionToString from "./transformCollectionToString.js";
+
+let collectionNotEmpty = transformCollectionToString([
+  "apple",
+  "grapes",
+  "strawberries",
+]);
+console.log(collectionNotEmpty);
+
+let collectionEmpty = transformCollectionToString([]);
```

Comment:
Same here.

```suggestion
let outputFromEmptyCollection = transformCollectionToString([]);
```

FIX: better use `const` if this constant will not change.